src/examples/design/icons/IconStyles.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { MD } from '@zendeskgarden/react-typography';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { ReactComponent as LeafStrokeIcon } from '@zendeskgarden/svg-icons/src/16/leaf-stroke.svg';
+import { ReactComponent as LeafFillIcon } from '@zendeskgarden/svg-icons/src/16/leaf-fill.svg';
+import { getColor } from '@zendeskgarden/react-theming';
+
+const StyledCol = styled(Col)`
+  text-align: center;
+  color: ${p => getColor('gray', 600, p.theme)};
+`;
+
+const Example = () => {
+  return (
+    <Grid>
+      <Row justifyContent="center">
+        <StyledCol size={3}>
+          <LeafStrokeIcon />
+          <MD>Stroke style (default)</MD>
+        </StyledCol>
+        <StyledCol size={3}>
+          <LeafFillIcon />
+          <MD>Filled Styled</MD>
```

Comment:
These words read a bit strange because [stroke](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke) and [fill](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill) are words used to describe attributes of SVG and words aren't typically used as past tense verbs. Also, it reads strange because one "style" is a noun, while the other is a past tense verb, "styled". I think _"Stroke (default)"_ and _"Fill"_ reads more clean than _"Stroke style (default)"_ and _"Filled Styled"_.

Reply:
@hzhu Major +1. I like your suggestion